tuna/miopen/miopen_tables.py

```diff
@@ -569,6 +569,37 @@ class BNBenchmark(BASE, BenchmarkMixin):
   config = Column(Integer, ForeignKey("bn_config.id"), nullable=False)
 
 
+class SolverAnalyticsResults(BASE):
+  """Table to store results from running SolverAnalytics"""
+  __tablename__ = "solver_analytics_results"
+
+  # each row in table must correspond to a unique convolution problem
+  __table_args__ = (UniqueConstraint("golden_v",
+                                     "filter",
+                                     "padding",
+                                     "stride",
+                                     "layout",
+                                     "precision",
+                                     "direction",
+                                     name="uq_idx"),)
+  
+  # columns definitions
+  golden_v = Column(Integer, nullable=False)
+  filter = Column(String(32), nullable=False)
+  padding = Column(String(32), nullable=False)
+  stride = Column(String(32), nullable=False)
+  dilation = Column(String(32), nullable=False)
+  layout = Column(String(8), nullable=False)
+  precision = Column(String(8), nullable=False)
+  direction = Column(String(1), nullable=False)
```

Comment:
are you saving 'F', 'B', 'W'? or 1,2,4? if its the ladder please make it an int.

Reply:
It's the former, F,B,W